File: skills/astra-toolkit/clients/typescript/examples/tables-drop-index.ts
import { DataAPIClient } from "@datastax/astra-db-ts";

// Get a database
const client = new DataAPIClient();
const database = client.db(process.env.ASTRA_DB_API_ENDPOINT!, {
  token: process.env.ASTRA_DB_APPLICATION_TOKEN!,
});

// Drop an index
(async function () {
  await database.dropTableIndex("**INDEX_NAME**");
})();

File: skills/astra-toolkit/clients/java/examples/tables-drop-index.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.databases.Database;

public class Example {

  public static void main(String[] args) {
    // Get a database
    Database database = new DataAPIClient(System.getenv("ASTRA_DB_APPLICATION_TOKEN")).getDatabase(System.getenv("ASTRA_DB_API_ENDPOINT"));

    // Drop an index
    database.dropTableIndex("**INDEX_NAME**");
  }
}
